src/animations.ts

```diff
@@ -539,25 +561,58 @@ function getSampleQuestionComponents() {
 }
 
 
+function onSampleQuestionSectionIntersection(entries: IntersectionObserverEntry[], audioSourceToAnimationNameMap: Record<string, AnimationName>) {
+    if (entries[0].isIntersecting) {
```

Comment:
I'm lacking context, just making sure there's always at least one entry?

Reply:
It should be exactly one since the observer is just tracking one element (the div for this section). I'll update this to safely check that an entry exists though to make sure it doesn't crash the page for the user :) 

And as a note: I'm sure we'll need to track more elements in the future as we add more animations. When that time comes, I think the cleanest option would be to set up a second observer so that we invoke a different callback for the different animations. The other option I see would be to use a single observer for everything, then invoke a wrapper function that looks at the element and determine which function to call based on the element.